libres/lib/enkf/analysis_config.cpp

```diff
@@ -154,12 +154,12 @@ int analysis_config_get_min_realisations(const analysis_config_type *config) {
     return config->min_realisations;
 }
 
-stringlist_type *
+std::vector<std::string>
 analysis_config_alloc_module_names(const analysis_config_type *config) {
```

Comment:
It has been a strong pattern: Functions with `_alloc_` in the name allocate memory and a corresponding `xxx_free` needs to be somewhere, with `std::vector<>` and value semantics I would suggest renaming the function e.g. `analysis_config_module_names()`